src/utils/date.ts

```diff
@@ -16,14 +16,14 @@ export const convertUtcTimestampToLocal = (utcTimestamp: number) => {
   return dateUTC.unix()
 }
 
-export const convertTimestampWithMoment = (utcTimestamp: number) => {
+export const faketimer = (utcTimestamp: number) => {
```

Comment:
`camelCase`

Reply:
These functions are not used anymore. I will remove them